docs/cn/instructions/quickstart-cn.md

```diff
@@ -0,0 +1,505 @@
+# 快速启动
+
+## 一、项目定位
+
+- EventMesh是一个动态的云原生事件驱动架构基础设施，它用于分离应用程序和后端中间件层，在业务层和基础结构层之间作为一个中间层的云原生的基础设施，它可以实现自动扩容以及对事件集中处理，还可以实现不同云原生下分布式的系统以及对消息的事件存储，后续还会支持更多的特性，比如云服务特性、数据实时计算等。
+- EventMesh系统支持多语言接入和多协议接入。因为在实际业务中，不同业务之间边界性高，但是本系统可以打通业务之间壁垒，拓宽业务场景。
+
+## 二、构建并部署event-store(RocketMQ)
+
+关于RocketMQ更多的细节：
+
+- 英文：https://github.com/apache/rocketmq/tree/master/docs/en
+- 中文：https://github.com/apache/rocketmq/tree/master/docs/cn
+
+### 2.1安装
+
+#### 依赖
+
+假设已经搭建完以下环境
+
+```
+64位操作系统，推荐使用Linux/Unix/Max;(Windows用户参阅下面指南)；
+64位JDK 1.8+；
+Maven 3.2.x；
```

Comment:
If we use binary version, we may don't need maven and git here. 
And I think we may don't need to introduce the detail of how to deploy the rocketmq, we can just provide a link to rocketmq-quickstart here https://rocketmq.apache.org/docs/quick-start/, because we may support other store plugin, and we couldn't introduce each plugin like this.

Reply:
`If we use binary version, we may don't need maven and git here.` But what if we don't use binary versions.
The second suggestion I will refer to your opinion.

Reply:
Maybe the `Maven 3.2.x` should in section `#### 构建源码`